src/wast-parser.y

```diff
@@ -1462,6 +1540,19 @@ ExprList join_exprs2(Location* loc, ExprList* expr1, Expr* expr2) {
   return result;
 }
 
+ExprList join_expr_lists(Location* loc, ExprList* expr1, ExprList* expr2) {
```

Comment:
Hm, this looks like it could just use `append_expr_list` twice. Though honestly the whole thing should be rewritten to C++-style, so this is mostly just nitpicking.